sdk/tools/cabman/CCFDATAStorageUnix.cxx

```diff
@@ -0,0 +1,168 @@
+/*
+ * PROJECT:     ReactOS cabinet manager
+ * LICENSE:     GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
+ * PURPOSE:     CCFDATAStorage class implementation for Linux/Unix
+ * COPYRIGHT:   Copyright 2017 Casper S. Hornstrup (chorns@users.sourceforge.net)
+ *              Copyright 2017 Colin Finck <mail@colinfinck.de>
+ *              Copyright 2018 Dmitry Bagdanov <dimbo_job@mail.ru>   
+ */
+#if !defined(_WIN32)
```

Comment:
So... this really isn't so much a Unix implementation as it is a standard C library one. Why don't we just delete the Win32 version and use this across the board? (I think that may have been the intent behind the Jira ticket)

Reply:
I thought about that ;) 
But then I decided to keep the initial intention to not use standard C library file access functions on Windows...

Reply:
exact.

Reply:
Well that will be the next pull request then.

Reply:
The problem with that approach is that what you propose now is not a natural step between the current version of the code and where we'd probably want it to be. E.g. changing if-CreateFile-else-fopen to fopen is much more straightforward than changing it to OpenFileForReading, and then back to the fopen that was already there before. That just seems like unnecessary code churn.
I'm open to discussion, but right now it seems to me like literally deleting all the stuff in the _WIN32 code path would get us closest to where we want to be.

Reply:
ThFabba, now I'm completely agree with you. That was my misunderstanding that the intention is to not use standard C file access functions on Windows. So I'm going to provide another pull request that uses standard C functions on all platforms.

Reply:
@dfbag7 For reference, if you push commits to the same branch they will end up in this pull request.
If you rewrite this branches history and then  force-push you will update this pull request.

Reply:
Otherwise, as the intent is totally different from original idea, perhaps should we close that PR for a brand new one? New code, new direction, new PR? :-)

Reply:
Whatever the process here is - I'm fine with either updating or creating a new PR - I conclude this is WIP, so I'll add the WIP tag.

Reply:
@dfbag7 Can you please make a basic performance evaluation after your changes? I am asking because I have experienced (mainly with STL) that portable code can lead to heavily degrading performance and this tool is already kinda slow, so I wouldn't want it to become even slower.